java/debugger.jpda/src/org/netbeans/modules/debugger/jpda/expr/EvaluatorVisitor.java

```diff
@@ -5512,7 +5512,7 @@ public Value getValue(Field field) {
 
         @Override
         public Map<Field, Value> getValues(List<? extends Field> list) {
-            List[] listByTypes = new List[types.length];
+            List<Field>[] listByTypes = new List[types.length];
```

Comment:
diamond on the right missing

Reply:
you are right, my mistake. I somehow saw it as `List<Field[]>`. Generic arrays are not possible in java.

Reply:
> you are right, my mistake. I somehow saw it as `List<Field[]>`. Generic arrays are not possible in java.

Hey @mbien , trust me, my eyes start to glaze over doing this stuff! 8-) For some reason, I continue to do it.. hehehe..